projects/components/src/select/select.component.scss

```diff
@@ -17,7 +17,7 @@
   display: flex;
   align-items: center;
   width: 100%;
-  height: 34px;
+  height: 36px;
```

Comment:
This seems like it could have more far reaching affects. have we checked all of the current selects that they're not being messed up by resizing?

Reply:
I spot checked several. 36px is correct and showing in the checked locations. I see no issues with overflow or misalignment.